server_manager/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This seems to match the original behavior. We should consider factoring out the common settings across projects at some point, like we have in outline-server.